ApeSwapFinance/frontend
# [CHORE] - Move React-query hooks to State folder

~## **Depends on:**~
~https://github.com/ApeSwapFinance/frontend/pull/137~

## ⛑️ **What was done:**
- Moved React-query hooks and types to `state` folder for organizational purposes, caching is also a type of state.

The agreed folder structure going forward is:

```
- src/state
  - bills
    - hooks
      - useGetSomething.ts
      - useUpdateSomethingElse.ts
     - types.ts
  - farms
    - hooks
      - useGetSomething.ts
      - useUpdateSomethingElse.ts
    - types.ts
 ```

### 🧪 **How to test:**
- Go to homepage `/`, should load without crashing.
- Go to `/liquidity-health`, password modal should verify correct password.
- Tokens should load, filter and sort without crashing.
- Go to `/liquidity-health/{CHAIN_ID}/{TOKEN_ADDRESS}`, should load without crashing.
